client/src/navigator/Drawer/Drawer.js

```diff
@@ -4,24 +4,258 @@ import {
+          >
+            <Pressable
+              style={{
+                // flex: 1,
+                alignItems: 'center',
+                justifyContent: 'center',
+                paddingVertical: 12,
+                paddingHorizontal: 10,
+                borderRadius: 10,
+                marginHorizontal: 10,
+                backgroundColor: '#F9F9F9',
+              }}
+              forceInset={{
+                top: 'always',
+                horizontal: 'never',
+              }}
+            >
+              <Text
+                fontWeight="regular"
+                color="gray.dark"
+                fontSize="sm"
+                textAlign="left"
+              >
+                Do you know an indigenous language that you would like to share
+                with the world?
+                <Text
+                  style={{
+                    fontFamily: 'GT_Haptik_bold',
+                  }}
+                >
+                  {' '}
+                  Become a contributor.
+                </Text>
+              </Text>
+              <StyledButton title="Apply Now" fontSize="sm" />
+            </Pressable>
+          </View>
+        ) : null}
+      </DrawerContentScrollView>
+      <View
+        style={{
+          marginTop: '8%',
+          marginBottom: '8%',
+          height: 1,
+          backgroundColor: '#EFEFEF',
+          width: '90%',
+        }}
+      />
+  
+      {/* Full View of Profile */}
+      <View>
+        
+        
+        {/* Photo from Google Auth goes here */}
+
+
+        {/* TEXT Profile View */}
+        <View
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            marginRight: 20,
+          }}
+          >
+          <Text
+            style={{
+              paddingLeft: 10,
+              fontFamily: 'GT_Haptik_bold',
+              fontSize: 20,
+            }}
+          >
+            Michael Scott
+          </Text>
+          <Text
+            style={{
+              fontSize: 15,
+              paddingLeft: 10,
+            }}
+          >
+            michaelscott@gmail.com
+          </Text>
+
+          
+        </View>
+
+        
+      </View>
+      
+      {/* DRAWER LOGOUT BUTTON */}
+      <View
+        style={{
+          marginTop: '8%',
+         
+          height: 1,
+          backgroundColor: '#EFEFEF',
+          width: '90%',
+        }}
+      />
+      <DrawerLogoutButton />
+      
+      
+    </>
   )
 }
 
 export default () => (
-  <Drawer.Navigator initialRouteName="Home" drawerContent={DrawerMenuContainer}>
-    <Drawer.Screen name="Home" component={TabNavigator} />
+  <Drawer.Navigator
+    drawerContentOptions={{
+      activeTintColor: 'black',
+      inactiveTintColor: 'black',
+      activeBackgroundColor: '#F9F9F9',
+      inactiveBackgroundColor: 'white',
+      itemStyle: { marginVertical: 2 },
+    }}
+    initialRouteName="Home"
+    drawerContent={DrawerMenuContainer}
+  >
```

Comment:
Amazing work with the styling for this component!!!

One thing to consider is how to modularize this code. There is a lot of repetition between the `Drawer.Screen` components (I'll refer to them as a `Tab`). One way to reduce this is to create a component in `client/src/components `that takes in props like the Drawer, title, number of units, and boolean indicating contributor, and renders one of the Tab.